#!/usr/bin/env bash

sudo python3 /usr/local/apps/cloud\ drive/modules/sync.py

exit